examples/cloud-provider-snapshots/snapshot.json

```diff
@@ -0,0 +1,38 @@
+{
+  "AWSTemplateFormatVersion" : "2010-09-09",
+  "Description" : "This template creates a cloud provider snapshot on the MongoDB Atlas API, this will be billed to your Atlas account.",
+  "Parameters" : {
+    "ProjectId" : {
+      "Type" : "String",
+      "Description" : "",
+      "ConstraintDescription" : ""
+    },
+    "ClusterName" : {
+      "Type" : "String",
+      "Description" : "",
+      "ConstraintDescription" : ""
+    }
+  },
+  "Mappings" : {},
+  "Resources" : {
+    "CloudProviderSnapshots": {
+      "Type": "MongoDB::Atlas::CloudProviderSnapshots",
+      "Properties": {
+        "ProjectId": { "Ref" : "ProjectId" },
+        "ClusterName": { "Ref" : "ClusterName" },
```

Comment:
Is `Cluster` merged yet? Maybe you can add an example with the `Cluster` resource too.

Reply:
Not yet, but if I do that, it returns an id of the cluster instead of the name of a cluster depending of the primaryIdentifier configured.

Reply:
Ah, gotcha